src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperOutput.cs

```diff
@@ -24,44 +24,32 @@ internal TagHelperOutput(string tagName)
             Attributes = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
         }
 
-        // Internal for testing
-        internal TagHelperOutput(string tagName, [NotNull] IDictionary<string, string> attributes)
-            : this(tagName, attributes, string.Empty)
-        {
-        }
-
         /// <summary>
         /// Instantiates a new instance of <see cref="TagHelperOutput"/>.
         /// </summary>
         /// <param name="tagName">The HTML element's tag name.</param>
         /// <param name="attributes">The HTML attributes.</param>
-        /// <param name="content">The HTML element's content.</param>
-        public TagHelperOutput(string tagName,
-                               [NotNull] IDictionary<string, string> attributes,
-                               string content)
+        public TagHelperOutput(string tagName, [NotNull] IDictionary<string, string> attributes)
         {
             TagName = tagName;
-            Content = content;
             Attributes = new Dictionary<string, string>(attributes, StringComparer.OrdinalIgnoreCase);
+            PreContent = string.Empty;
```

Comment:
what does "unusable" mean?  `null` values are written out identically to `string.Empty` still; aren't they?